codeSnippets/snippets/full-stack-task-manager/README.md

```
@@ -0,0 +1,19 @@
+This is a Kotlin Multiplatform project targeting Android, iOS, Desktop, Server.
+
+* `/composeApp` is for code that will be shared across your Compose Multiplatform applications.
+  It contains several subfolders:
+  - `commonMain` is for code that’s common for all targets.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)